From d2b45a9bdd07d0db19eb7dd53bc0671f89687850 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 13 Dec 2020 04:20:08 -0500
Subject: [PATCH 01/80] Revert "Use ext4 share duplicate blocks, it is sadly
 needed to attain acceptable partition size, because of duplication made by
 apexes"

This reverts commit 30071d042053b67e4ec6d7332ae187d6cd4542db.
---
 board-base.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board-base.mk b/board-base.mk
index 0da1061..4739954 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -4,3 +4,4 @@ TARGET_EXFAT_DRIVER := exfat
 DEVICE_FRAMEWORK_MANIFEST_FILE := device/phh/treble/framework_manifest.xml
 
 BOARD_ROOT_EXTRA_FOLDERS += bt_firmware sec_storage efs
+BOARD_EXT4_SHARE_DUP_BLOCKS := false
-- 
2.25.1

